tests/snapshots/test_app/test_ask_vision/client1/result.json

```diff
@@ -60,16 +57,15 @@
                                 "text": "Are interest rates high?",
                                 "type": "text"
                             },
-                            {
-                                "text": "Financial Market Analysis Report 2023-6.png: 3</td><td>1</td></tr></table> Financial markets are interconnected, with movements in one segment often influencing others. This section examines the correlations between stock indices, cryptocurrency prices, and commodity prices, revealing how changes in one market can have ripple effects across the financial ecosystem.Impact of Macroeconomic Factors Impact of Interest Rates, Inflation, and GDP Growth on Financial Markets 5 4 3 2 1 0 -1 2018 2019 -2 -3 -4 -5 2020 2021 2022 2023 Macroeconomic factors such as interest rates, inflation, and GDP growth play a pivotal role in shaping financial markets. This section analyzes how these factors have influenced stock, cryptocurrency, and commodity markets over recent years, providing insights into the complex relationship between the economy and financial market performance. -Interest Rates % -Inflation Data % GDP Growth % :unselected: :unselected:Future Predictions and Trends Relative Growth Trends for S&P 500, Bitcoin, and Oil Prices (2024 Indexed to 100) 2028 Based on historical data, current trends, and economic indicators, this section presents predictions ",
-                                "type": "text"
-                            },
                             {
                                 "image_url": {
-                                    "detail": "auto",
                                     "url": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z/C/HgAGgwJ/lK3Q6wAAAABJRU5ErkJggg=="
                                 },
                                 "type": "image_url"
+                            },
+                            {
```

Comment:
This looks like a larger change, but is actually just an ordering difference, where the text sources come after the image. That was due to how Prompty breaks up multi-part messages, and I don't think it would affect answer quality.